kolibri/plugins/coach/assets/src/views/coach-exercise-render-page/attempt-log-list.vue

```diff
@@ -1,44 +1,41 @@
 <template>
 
-  <div class="answer-history">
+  <div class="attempt-log-list">
     <h3 class="header">{{ $tr('header') }}</h3>
 
     <ul class="history-list">
       <template v-for="(attemptLog, index) in attemptLogs">
-        <li v-if="index === 0">
-          <p class="item">
-            {{ daysElapsedText(attemptLog.daysElapsed) }}
-          </p>
-        </li>
+        <!--p v-if="index === 0" class="item">
+          TODO modify elapsed-time to do Days only?
```

Comment:
Yeah, can also tweak the component to not need to update `now` so often if the `smallestUnit` or `resolution` property is relatively high.

I think `smallestUnit` is the most explanatory name for the prop :)